.github/workflows/android-perf.yml

```diff
@@ -132,10 +132,10 @@ jobs:
       matrix: ${{ fromJson(needs.set-parameters.outputs.benchmark_configs) }}
       fail-fast: false
     with:
-      runner: linux.2xlarge.memory
+      runner: linux.4xlarge.memory
```

Comment:
Locally on devserver, quantization is taking "INFO:root:Time for quantizing: 1203.9422521591187", but on CI it's 4x slower, bump up to use the 4x runner.